webapp/WEB-INF/templates/admin/plugins/mylutece/modules/users/manage_localuserroles.html

```diff
@@ -0,0 +1,42 @@
+<#include "localusermanagement_tabs.html" />
+<#assign headerIcon>
+    <@icon style='user' />
+</#assign>
+<@box>
+    <@boxHeader title='${headerIcon}&nbsp; ${localuser.givenName} ${localuser.familyName}' boxTools=true>
+    </@boxHeader>
+    <@boxBody>
+        <@localusermanagement tab="localuserrole" />
+        <@messages infos=infos />
+        <div class="clearfix"></div>
+        <@tform name='action_doassignlocaluserrole'
+            action='jsp/admin/plugins/mylutece/modules/users/ManageLocalUserRoles.jsp'>
+            <input type="hidden" id="id_localuser" name="id_localuser" value="${id_localuser}" />
+            <@row>
+                <@columns>
+                    <@listGroup>
+                        <#list mylutece_role_list as role>
+                            <#if role.role !="none">
```

Comment:
it is not a good practice to create a "none" role to represent an empty list

Reply:
impossible.